tools/make/lib/lint/typescript/Makefile

```diff
@@ -32,113 +32,65 @@ include $(TOOLS_MAKE_LIB_DIR)/lint/typescript/dtslint.mk
-ifeq ($(TYPESCRIPT_LINTER), tslint)
-	$(QUIET) NODE_ENV="$(NODE_ENV)" NODE_PATH="$(NODE_PATH)" $(MAKE) -f $(this_file) tslint-src
-endif
-
-.PHONY: lint-typescript-src
-
-#/
-# Lints TypeScript test files.
-#
-# ## Notes
-#
-# -   This rule is useful when wanting to glob for TypeScript test files (e.g., lint all TypeScript test files for a particular package).
-#
-# @param {string} [TYPESCRIPT_TESTS_FILTER] - file path pattern (e.g., `.*/math/base/special/abs/.*`)
-# @param {*} [FAST_FAIL] - flag indicating whether to stop linting upon encountering a lint error
-#
-# @example
-# make lint-typescript-tests
-#
-# @example
-# make lint-typescript-tests TYPESCRIPT_TESTS_FILTER=".*/math/base/special/abs/.*"
-#/
-lint-typescript-tests:
-ifeq ($(TYPESCRIPT_LINTER), tslint)
-	$(QUIET) NODE_ENV="$(NODE_ENV)" NODE_PATH="$(NODE_PATH)" $(MAKE) -f $(this_file) tslint-tests
-endif
-
-.PHONY: lint-typescript-tests
-
-#/
-# Lints TypeScript examples files.
+# Lints TypeScript declaration files.
 #
 # ## Notes
 #
-# -   This rule is useful when wanting to glob for TypeScript examples files (e.g., lint all TypeScript examples files for a particular package).
+# -   This rule is useful when wanting to glob for TypeScript declaration files (e.g., lint all TypeScript declaration files for a particular package).
 #
-# @param {string} [TYPESCRIPT_EXAMPLES_FILTER] - file path pattern (e.g., `.*/math/base/special/abs/.*`)
+# @param {string} [TYPESCRIPT_DECLARATIONS_FILTER] - file path pattern (e.g., `.*/math/base/special/abs/.*`)
 # @param {*} [FAST_FAIL] - flag indicating whether to stop linting upon encountering a lint error
 #
 # @example
-# make lint-typescript-examples
+# make lint-typescript-declarations
 #
 # @example
-# make lint-typescript-examples TYPESCRIPT_EXAMPLES_FILTER=".*/math/base/special/abs/.*"
+# make lint-typescript-declarations TYPESCRIPT_DECLARATIONS_FILTER=".*/math/base/special/abs/.*"
 #/
-lint-typescript-examples:
-ifeq ($(TYPESCRIPT_LINTER), tslint)
-	$(QUIET) NODE_ENV="$(NODE_ENV)" NODE_PATH="$(NODE_PATH)" $(MAKE) -f $(this_file) tslint-examples
+lint-typescript-declarations:
+ifeq ($(TYPESCRIPT_DECLARATIONS_LINTER), eslint)
+	$(QUIET) NODE_ENV="$(NODE_ENV)" NODE_PATH="$(NODE_PATH)" $(MAKE) -f $(this_file) eslint-ts-declarations
 endif
 
-.PHONY: lint-typescript-examples
+.PHONY: lint-typescript-declarations
 
 #/
-# Lints TypeScript declaration files.
+# Lints TypeScript declaration test files.
 #
 # ## Notes
 #
-# -   This rule is useful when wanting to glob for TypeScript declaration files (e.g., lint all TypeScript declaration files for a particular package).
+# -   This rule is useful when wanting to glob for TypeScript declaration test files (e.g., lint all TypeScript declaration test files for a particular package).
 #
-# @param {string} [TYPESCRIPT_DECLARATIONS_FILTER] - file path pattern (e.g., `.*/math/base/special/abs/.*`)
+# @param {string} [TYPESCRIPT_DECLARATIONS_TESTS_FILTER] - file path pattern (e.g., `test-*.js`)
```

Comment:
@Planeshifter Why change the filter example here? I am not sure `test-*.js` works here. Same thing for L86.

Reply:
Also, this pattern doesn't match our filename convention (uses hyphens, which we don't generally do).

Reply:
I've already updated this.